plugin-server/tests/e2e.test.ts

```diff
@@ -121,6 +121,29 @@ describe('e2e', () => {
             expect(testConsole.read()).toEqual([['processEvent'], ['onEvent', 'custom event']])
         })
 
+        test('buffered event captured, processed, ingested', async () => {
```

Comment:
Note we're lacking a true e2e test here which would ensure we send an event to posthog -> it gets buffered -> it's eventually processed correctly.

Reply:
Removed this one and added a new, fuller test as `e2e.buffer.test.ts`